packages/esm-patient-test-results-app/src/hiv/concepts.ts

```diff
@@ -0,0 +1,58 @@
+const concepts = {
+  display: 'Bloodwork',
+  concept: 'ConvSet', // convienence set
+  obs: [],
```

Comment:
Given that the set itself will not have obs, do we need the above `obs: []`?

Reply:
No, we don't need to pass a key if it is empty